cmd/sqlflowserver/main.go

```diff
@@ -60,10 +69,12 @@ func start(modelDir, caCrt, caKey string, port int, isArgoMode bool) {
 			os.Mkdir(modelDir, os.ModePerm)
 		}
 	}
-	if isArgoMode {
-		proto.RegisterSQLFlowServer(s, server.NewServer(server.SubmitWorkflow, modelDir))
+	if workflowBackend == "" {
+		proto.RegisterSQLFlowServer(s, server.NewServer(sf.RunSQLProgram, modelDir, ""))
 	} else {
-		proto.RegisterSQLFlowServer(s, server.NewServer(sf.RunSQLProgram, modelDir))
+		proto.RegisterSQLFlowServer(s, server.NewServer(func(sqlProgram string, modelDir string, session *pb.Session) *pipe.Reader {
+			return server.SubmitWorkflow(sqlProgram, modelDir, session, workflowBackend)
```

Comment:
Read `workflowBackend` from env so that we don't have to change `SubmitWorkflow` arguments.